.golangci.yml

```diff
@@ -1,5 +1,6 @@
 run:
   timeout: 10m
+  go: '1.17'
```

Comment:
1.18?

Reply:
golangci don't support some linter in go 1.18, so now run in go1.17.

When we want to use the feature of go 1.18, we can change it.